pkiinit/pkiInit.go

```diff
@@ -29,31 +32,47 @@ type exitCode struct {
 	exit
```

Comment:
this could be simplified by removing the exit object and the parameter in newExit()
type exitCode struct {
}
func newExit() exit {
	return &exitCode{}
}